Competition2018/src/main/java/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -46,6 +50,32 @@ public DriveSubsystem(CommonLibFactory factory, XPropertyManager propManager) {
         masterTalons = new HashMap<XCANTalon, BaseDriveSubsystem.MotionRegistration>();
         masterTalons.put(leftMaster, new MotionRegistration(0, 1, -1));
         masterTalons.put(rightMaster, new MotionRegistration(0, 1, 1));
+        
+        
+        leftMaster.createTelemetryProperties("LeftDriveMaster");
+        rightMaster.createTelemetryProperties("RightDriveMaster");
+        
+        leftTicksPerFiveFeet = propManager.createPersistentProperty("leftDriveTicksPer5Feet", 0);
```

Comment:
Per... 5 feet? That seems odd. If I were measuring ticks I'd do something longer like 10 feet anyway, given that small error will accumulate.

Reply:
It's a balance - I chose a measurement I thought we'd be able to do quickly in a pinch, potentially in a cramped district event.

While we're at 99, we can push it 10 feet and just divide by 2.

Reply:
😆 At that point why don't we just enter it out of 1 foot?

Reply:
It's just a gentle reminder for people to take longer measurements if possible. If somebody who understands the DS console well (e.g. the drive team) needs to recalibrate the encoders, the fact that the variable has a long (but achievable) measurement will hopefully remind them to get some distance.